src/link.coffee

```diff
@@ -29,6 +29,8 @@ class Link extends Command
     {callback} = options
     options = @parseOptions(options.commandArgs)
 
+    options.argv._[0] = if options.argv._[0] then String(options.argv._[0])
```

Comment:
Instead of assigning back to this array, what do you think about pulling out a variable that incorporates defaulting and stringifying into its own line.

Something like:

```coffee
packagePath = options.argv._[0]?.toString() ? '.'
linkPath = path.resolve(process.cwd(), packagePath)
```